ansible/deploy_cluster_test_plugins.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2024 Savoir-faire Linux, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+---
+# ansible_repository_path can be set in the inventory or via the argument "-e"
+- import_playbook: "{{ ansible_repository_path | default('../../../..') }}/playbooks/cluster_setup_debian.yaml"
+
+- name: Configure the cluster for Cockpit plugins testing
+  hosts:
+      - cluster_machines
+      - localhost
+  vars_files:
+    - roles/cockpit_plugins_cluster_setup/vars/main.yml
+  tasks:
+      - name: Check if cockpit is installed
+        command: which cockpit-bridge
+        register: cockpit_status
+        when: inventory_hostname in groups['cluster_machines']
+
+      - name: Cockpit plugins installation (localhost)
+        include_tasks: roles/cockpit_plugins_cluster_setup/tasks/fetch_plugins.yml
+        when: inventory_hostname == 'localhost'
+
+      - name: Cockpit plugins installation (localhost)
+        include_tasks: roles/cockpit_plugins_cluster_setup/tasks/build_plugins.yml
+        when: inventory_hostname == 'localhost'
+
+      - name: Cockpit plugins installation (cluster machines)
+        become: true
+        block:
+          - include_tasks: roles/cockpit_plugins_cluster_setup/tasks/deploy_plugins_cluster.yml
+        when: inventory_hostname in groups['cluster_machines'] and cockpit_status.rc == 0
+
+        # For Selenium tests purposes
+      - name: Shutdown one of the host
+        block:
+          - command: "shutdown now"
```

Comment:
How do you reboot the nuc3 when the test is ended?

Reply:
The host can be reboot using wakeonlan with the MAC address. This information has been added to the readme.